contracts/serrano/PKPHelperV2.json

```diff
@@ -0,0 +1,340 @@
+[
+  {
+    "inputs": [
```

Comment:
Do we need to update the `serrano` abi? will this be getting deployed on the Serrano network?